common/build/uboot.mk

```diff
@@ -144,6 +151,13 @@ $(UBOOT_BIN): $(UBOOTENVSH) | $(UBOOT_COLLECTION) $(UBOOT_OUT)
 		install -D $(UBOOT_COLLECTION)/flash.bin $(UBOOT_BIN); \
 		UBOOT_BD_NAME=`echo u-boot.$$UBOOT_CONFIG | sed 's/_defconfig//'`; \
 		install -D $(UBOOT_COLLECTION)/flash.bin $(PRODUCT_OUT)/preboot/$$UBOOT_BD_NAME; \
+		if [ "$(UBOOT_SIGN)" = "true" ]; then \
+			echo "Signing $$UBOOT_BD_NAME"; \
+			cd $(UBOOT_OUT); \
+			$(ANDROID_BUILD_TOP)/$(UBOOT_IMX_PATH)/uboot-imx/sign_hab_imx8m.sh; \
+			cd -; \
+			install -D $(UBOOT_OUT)/signed_flash.bin $(PRODUCT_OUT)/preboot/signed_$$UBOOT_BD_NAME; \
```

Comment:
sorry, one last change, please don't name it signed_* cause otherwise it will break the upgrade script. If you sign your bootloader, it means you don't need the unsigned version. Plus the signed version will boot on an unfused device anyway.

Reply:
Good point, I didn't notice it as we do not use preboot image anymore ;)